containers/web/Dockerfile

```diff
@@ -39,23 +39,31 @@ RUN awk '/PATH=/ { print; print "MAILTO=\"\""; next}1' /etc/crontab > /tmp/cront
 RUN awk '/LOGNAME=root/ { print; print "MAILTO=\"\""; next}1' /etc/anacrontab > /tmp/anacrontab && mv /tmp/anacrontab /etc/anacrontab
 
 # Setup persistent environment variables
-ENV XIBO_VERSION=1.8.0-alpha3 XMR_HOST=xibo-cms-xmr
-ENV XIBO_SMTP_SERVER=smtp.gmail.com:587 XIBO_SMTP_USERNAME=none XIBO_SMTP_PASSWORD=none XIBO_SMTP_USE_TLS=YES XIBO_SMTP_USE_STARTTLS=YES XIBO_SMTP_REWRITE_DOMAIN=gmail.com XIBO_SMTP_HOSTNAME=none XIBO_SMTP_FROM_LINE_OVERRIDE=YES
+ENV CMS_VERSION=1.8.0-alpha3 XMR_HOST=xmr CMS_DB_VERSION=123
+ENV CMS_DATABASE_HOST=mysql CMS_DATABASE_USERNAME=cms CMS_DATABASE_PASSWORD=none CMS_DATABASE_PORT=3306 CMS_DATABASE_NAME=cms
+ENV CMS_SERVER=smtp.gmail.com:587 CMS_SMTP_USERNAME=none CMS_SMTP_PASSWORD=none CMS_SMTP_USE_TLS=YES CMS_SMTP_USE_STARTTLS=YES CMS_SMTP_REWRITE_DOMAIN=gmail.com CMS_SMTP_HOSTNAME=none CMS_SMTP_FROM_LINE_OVERRIDE=YES
 
 # Expose port 80
 EXPOSE 80
 
 # Pull in the release archive for the environment
-ADD https://github.com/xibosignage/xibo-cms/releases/download/${XIBO_VERSION}/xibo-cms-${XIBO_VERSION}.tar.gz /var/www/xibo-cms.tar.gz
+RUN mkdir -p /var/www/cms && curl -o cms.tar.gz -SL https://github.com/xibosignage/xibo-cms/releases/download/${CMS_VERSION}/xibo-cms-${CMS_VERSION}.tar.gz \
+	&& tar --strip=1 -xzf cms.tar.gz -C /var/www/cms \
+	&& rm cms.tar.gz
 ADD https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh  /usr/local/bin/wait-for-it.sh
+ADD settings.php-template /var/www/cms/web/settings.php
 ADD settings.php-template /tmp/settings.php-template
```

Comment:
One copy of the settings file is copied in for CI mode. I can't avoid doing that. 

I could pass the SECRET_KEY in as a variable. The reason I'm not is that what it's set to doesn't really matter, only that it's relatively stable (ie doesn't change on every request) and that it's random. Passing it in is asking for users to set dumb defaults which reduces the quality of the nonces it's used to generate.